### DISCUSSION TITLE
Error with Discord Notify

### DISCUSSION THREAD

**Discussion by jdaraver2011** [swar/Swar-Chia-Plot-Manager#803] [Q&A]
I have 4 Linux systems that the Discord Webook works without issues but when setting the option to true on Windows 10 I get an error. 

Traceback (most recent call last):
  File "C:\SWAR\Swar-Chia-Plot-Manager\manager.py", line 30, in <module>
    start_manager()
  File "C:\SWAR\Swar-Chia-Plot-Manager\plotmanager\library\utilities\commands.py
", line 39, in start_manager
    test_configuration(chia_location=chia_location, notification_settings=notifi
cation_settings,
  File "C:\SWAR\Swar-Chia-Plot-Manager\plotmanager\library\utilities\configurati
on.py", line 16, in test_configuration
    raise MissingImportError('Failed to find import "discord_notify". Be sure to
 run "pip install -r '
plotmanager.library.utilities.exceptions.MissingImportError: Failed to find impo
rt "discord_notify". Be sure to run "pip install -r requirements-notification.tx
t".

**Comment by Lyushen** [comment]
Can you read? There is literally telling you what to do... Last sentence.. Just read it.

**Reply by jdaraver2011** [reply]
Didn't realize they added that step with the new install since the older boxes didn't require that to work. Thanks
